routes/web.php

```diff
@@ -24,4 +24,9 @@
     Route::resource('meals', 'MealsController');
     Route::resource('orders', 'OrdersController');
     Route::resource('menu', 'MenuController');
+
+    //email/password registration 
```

Comment:
I thought about this but then decided to be explicit with the routes since its auth and you already have some routes available which I used as well . I wanted to avoid the confusion of assuming say create method handles signups/ registration only for one to find out its only the custom registration that implements this and the google signup uses something else. Unless you feel this should be moved to a separate controller ?

Reply:
Yeah, I agree with what you are saying. I'm thinking of a refactor of auth and separating it into Social and Local auth. Something along the lines of:

**GoogleAuthController** -> handles logic around login/signup with Google
**PasswordLoginController** -> handles login with email & password combo
**UsersController** -> a resource controller to register, view, edit, etc. users
**AddUserJob** -> used by both _GoogleAuthController_ and _UsersController_ to create user accounts

What are your thoughts?

Reply:
This should work fine